### DISCUSSION TITLE
News Cache [2023-06-26]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#3011] [Announcements]
### 🚢  Ships

- [Svelte 4](https://svelte.dev/blog/svelte-4) stable release
- [Improvements and fixes](https://vercel.com/changelog/improvements-and-fixes), including…
    - [Draft Mode](https://vercel.com/docs/frameworks/nextjs#draft-mode) available in the Vercel toolbar
    - [Skew Protection](https://vercel.com/docs/concepts/deployments/skew-protection) to eliminate version skew between web clients and servers
    - Storage transfers between Hobby and Pro
    - The option to disable [ignore build step script](https://vercel.com/docs/concepts/projects/overview#ignored-build-step) when rebuilding or promoting a deployment, forcing the build to happen
    - New `VERCEL_BRANCH_URL` system environment variable gives access to a deployment’s [Git branch alias](https://vercel.com/docs/concepts/deployments/generated-urls#url-with-git-branch) in your code
- [Azure CosmosDB integration now available](https://vercel.com/changelog/azure-cosmosdb-integration-now-available)
- [Markdown support for comments on Preview Deployments](https://vercel.com/changelog/markdown-support-for-comments-on-preview-deployments)

### 💡 Inspiration

- [How Neo Financial cut time spent on infrastructure admin by 50%](https://vercel.com/customers/neo-financial)
- [Incrementally adopting Next.js at one of Europe's fastest growing brands](https://vercel.com/customers/incrementally-adopting-next-js-at-one-of-europes-fastest-growing-brands)

### 📚 Resources

- [An Introduction to Streaming on the Web](https://vercel.com/blog/an-introduction-to-streaming-on-the-web)
- [Next.js App Router Update](https://nextjs.org/blog/june-2023-update)
- [Novel](https://vercel.com/templates/next.js/novel) AI-powered WYSYWIG editor template
- [LaunchDarkly Integration example](https://vercel.com/templates/next.js/feature-flag-launchdarkly) template
- Tutorial: [Using Images in Next.js](https://twitter.com/leeerob/status/1673020151971684352?s=20)
- Tutorial: [Incrementally adopt the Next.js App Router](https://twitter.com/leeerob/status/1671650779676168192?s=20)

### ⚠️ FYI

_Legacy DNS config update:_ Some legacy IP addresses are being deprecated. If you have a domain configured to work with Vercel which points to one of them it will read as “Invalid Configuration”. All you have to do is update any affected records so they point to 76.76.21.21 instead. Domains still pointing to affected IP addresses will stop functioning after August 15, 2023.

**Comment by Anugrah123456** [comment]
I love you
